src/main/java/moze_intel/projecte/gameObjs/gui/AbstractCondenserScreen.java

```diff
@@ -39,9 +40,40 @@ protected void renderBg(@NotNull PoseStack matrix, float partialTicks, int x, in
 	@Override
 	protected void renderLabels(@NotNull PoseStack matrix, int x, int y) {
 		//Don't render title or inventory as we don't have space
-		long toDisplay = Math.min(menu.displayEmc.get(), menu.requiredEmc.get());
-		Component emc = TransmutationEMCFormatter.formatEMC(toDisplay);
-		this.font.draw(matrix, emc, 140, 10, 0x404040);
+		switch (ProjectEConfig.client.condenserEmcDisplayMode.get()) {
+		//Normal Old Rendering
+		default:
+			long toDisplay = Math.min(menu.displayEmc.get(), menu.requiredEmc.get());
+			Component emc = TransmutationEMCFormatter.formatEMC(toDisplay);
+			this.font.draw(matrix, emc, 140, 10, 0x404040);
+		break;
+			
+		//Turns green and shows total value if Stored > Required
+		case 1:
+			toDisplay = menu.displayEmc.get();
+			int displayColor = 0x404040;
+			if (toDisplay > menu.requiredEmc.get())
+				displayColor = 0x008000;
+			emc = TransmutationEMCFormatter.formatEMC(toDisplay);
+			this.font.draw(matrix, emc, 140, 10, displayColor);
+		break;
+		
+		case 2:
+			toDisplay = Math.min(menu.displayEmc.get(), menu.requiredEmc.get());
+			emc = TransmutationEMCFormatter.formatEMC(toDisplay);
+			this.font.draw(matrix, emc, 140, 10, 0x404040);
+			
+			if (menu.displayEmc.get() > menu.requiredEmc.get()) {
+				int xOff = 140 + this.font.width(emc);
+				emc = TransmutationEMCFormatter.formatEMC(menu.displayEmc.get());
+				if ((xOff + this.font.width(" / " + emc.getString())) < this.getXSize()) {
+					this.font.draw(matrix, " / " + emc.getString(), xOff, 10, 0x008000);
+				} else {
+					this.font.draw(matrix, " / Overflow", xOff, 10, 0x008000);
```

Comment:
All of this should be done via translation components defined in `PELang` and `PELangProvider`.

Reply:
Ah Right. I'll fix that.

Reply:
How should i handle the `" / "` additions? Should i add translation keys for that too?

Reply:
Yes that way for en_ud it can reverse the order of things properly.